AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/UpdateBookingStatusTests.cs

```diff
@@ -458,7 +458,57 @@ public async Task UpdateBookingStatus_returns_false_response_when_conference_exi
         }
 
         [Test]
-        public void UpdateBookingStatus_throws_when_bookings_api_exception()
+        public async Task UpdateBookingStatus_returns_failed_status_response_when_there_is_exception()
```

Comment:
Maybe change the name as it sounds like it should not return a success HTTP status code. Not just set the status flag.

Reply:
fixed test case title to include when you create a booking with exception, the result is back is ok but with failed booking status. (functionality requirement)